packages/datetime/src/dateRangePicker.tsx

```diff
@@ -370,15 +404,26 @@ export class DateRangePicker extends AbstractComponent<IDateRangePickerProps, ID
         Utils.safeInvoke(this.props.onHoverChange, dateRange, day, boundary);
     };
 
-    private handleDayMouseLeave = (day: Date, modifiers: DayPicker.DayModifiers) => {
+    private handleDayMouseLeave = (
+        day: Date,
+        modifiers: ReactDayPicker.DayModifiers,
+        e: React.MouseEvent<HTMLDivElement>,
+    ) => {
+        Utils.safeInvoke(this.props.dayPickerProps.onDayMouseLeave, day, modifiers, e);
```

Comment:
nit: should add whitespace below this line for consistency with other functions around this one. Not necessary before merging though.